src/api/search.js

```diff
@@ -1,31 +1,39 @@
 import { cachedFetch, buildMwApiUrl } from 'utils'
 
 export const search = (lang, term) => {
+  // fulltext search
   const params = {
     action: 'query',
-    prop: ['description', 'pageimages', 'pageprops'].join('|'),
+    prop: 'pageimages',
```

Comment:
`description` and `pageprops` are for the page item, since changing to `fulltext`, it consider the search response, all we need to get from the prop is only `pageimages`

confirm that we need both `generator` (gs* key used) and `list` (sr* key used)